### PR TITLE
Applied global level filters

### PR DESCRIPTION
**Pull request by agorthi-akamai**
## Description 📝

Highlight the Pull Request's context and intentions.

## Changes  🔄

List any change(s) relevant to the reviewer.

- ...
- ...

## Target release date 🗓️

Please specify a release date (and environment, if applicable) to guarantee timely review of this PR. If exact date is not known, please approximate and update it as needed.

## Preview 📷

**Include a screenshot or screen recording of the change.**

:lock: Use the [Mask Sensitive Data](https://cloud.linode.com/profile/settings) setting for security.

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
| 📷 | 📷 |

### Prerequisites

(How to setup test environment)

- ...
- ...

### Reproduction steps

(How to reproduce the issue, if applicable)

- [ ] ...
- [ ] ...

### Verification steps

(How to verify changes)

- [ ] ...
- [ ] ...

<details>
<summary> Author Checklists </summary>

## As an Author, to speed up the review process, I considered 🤔

*Check all that apply*

- [ ] Use React components instead of HTML Tags
- [ ] Proper naming conventions like cameCase for variables & Function & snake_case for constants
- [ ] Use appropriate types & avoid using "any"
- [ ] No type casting & non-null assertions
- [ ] Adding a [changeset](https://github.com/linode/manager/blob/develop/docs/CONTRIBUTING.md#writing-a-changeset)
- [ ] Providing/Improving test coverage
- [ ] Use sx props to pass styles instead of style prop
- [ ] Add JSDoc comments for interface properties & functions
- [ ] Use strict equality (===) instead of double equal (==)
- [ ] Use of named arguments (interfaces) if function argument list exceeds size 2
- [ ] Destructure the props
- [ ] Keep component size small & move big computing functions to separate utility
- [ ] 📱 Providing mobile support

<br/>

- [ ] I have read and considered all applicable items listed above.

## As an Author, before moving this PR from Draft to Open, I confirmed ✅

- [ ] All unit tests are passing
- [ ] TypeScript compilation succeeded without errors
- [ ] Code passes all linting rules

</details>

---

## Commit message and pull request title format standards

> **Note**: Remove this section before opening the pull request
**Make sure your PR title and commit message on squash and merge are as shown below**

`<commit type>: [JIRA-ticket-number] - <description>`

**Commit Types:**

- `feat`: New feature for the user (not a part of the code, or ci, ...).
- `fix`: Bugfix for the user (not a fix to build something, ...).
- `change`: Modifying an existing visual UI instance. Such as a component or a feature.
- `refactor`: Restructuring existing code without changing its external behavior or visual UI. Typically to improve readability, maintainability, and performance.
- `test`: New tests or changes to existing tests. Does not change the production code.
- `upcoming`: A new feature that is in progress, not visible to users yet, and usually behind a feature flag.

**Example:** `feat: [M3-1234] - Allow user to view their login history`

---

### CODE REVIEW

File: packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts [hunk 1]
```diff
@@ -283,11 +301,26 @@ describe('Integration Tests for Applied Filters', () => {
...
+        .should('have.text', 'PostgreSQL');
+
       cy.get(`[data-qa-value="Region US, Chicago, IL"]`)
-      cy.get(`[data-qa-value="Node Type Primary"]`)
-      cy.get(`[data-qa-value="Database Clusters ${databaseMock.label}"]`)
-      cy.get(`[data-qa-value="Database Clusters ${extendDatabaseMock.label}"]`)
+        .should('be.visible')
+        .should('have.text', 'US, Chicago, IL');
+
+      cy.get(`[data-qa-value="Node Type ${'Primary'}"]`)
```
Comment (ankita-akamai) [inline packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts]: @nikhagra-akamai this is going to change, right?

Comment (agorthi-akamai) [reply packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts]: Automation implemented based on the latest code. Any future changes will be incorporated and updated accordingly.

Comment (venkymano-akamai) [inline packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts]: @agorthi-akamai , here no need for back ticks right

Comment (agorthi-akamai) [reply packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-applied-filters.spec.ts]: @venkymano-akamai , i have addressed  it :)

File: packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-resource-limit.spec.ts [hunk 2]
```diff
@@ -0,0 +1,290 @@
...
+ * This test suite validates the behavior of the DBaaS (Database as a Service) Dashboard,
+ * specifically focusing on how the system handles the maximum resource selection limit
+ * for database clusters. It contains two main test cases:
+ * 
+ * Ticket: DI-21897
+ * 
+ * 1. **Max Resource Limit Reached**:
+ *    - Verifies that when the maximum number of database clusters (1) is selected,
+ *      the "Select All" option is hidden and only the available clusters (up to the limit)
+ *      are shown in the `Database Clusters` dropdown.
```
Comment (ankita-akamai) [inline packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-resource-limit.spec.ts:11]: clusters exceeding the selection limit will be disabled, but they will still be visible in dropdown..

Comment (agorthi-akamai) [reply packages/manager/cypress/e2e/core/cloudpulse/cloudpulse-resource-limit.spec.ts:11]: updated

**Review by nikhagra-akamai — APPROVED** [review]

**Review by venkymano-akamai — APPROVED** [review]

**Pull request merged**